classes/person.rb

```diff
@@ -0,0 +1,19 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age:, name: 'Unknown', parent_permission: true)
```

Comment:
- [ ] you passed in an age parameter to your initialize constructor function like this `age:`, doing this translates to `age: age` meaning you are setting a default value with the variable name `age` but there is no where in your code that age is defined as as such this is an error in your code. see the code snippet below for how it should be
- code snippet
```ruby
def initialize(age, name = "Unknown", parent_permission = true)
```